addons/binding/org.openhab.binding.konnected/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+	http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+	<config-description uri="thing-type:konnected:module">
+		<parameter-group name="actions">
+			<label>Actions</label>
+			<description></description>
+		</parameter-group>
+
+		<parameter name="authToken" type="text" required="true">
+			<label>Auth_Token</label>
+			<description>This is the Authority Token used to communicate with the Konnected Module.</description>
+		</parameter>
+		<parameter name="hostAddress" type="text">
+			<label>OpenHAB Address</label>
+			<description> The host address and port of openhab: example 192.168.1.180:8080.  If blank the binding will auto determine the address based on the address and port the server is bound to.</description>
+			<advanced>true</advanced>
+		</parameter>
+		<parameter name="callBackPath" type="text">
+			<label>OpenHAB CallBack</label>
+			<description> The callback path for where the module can send status updates. If left blank the binding will default to /Konnected/{ThingId}</description>
+			<advanced>true</advanced>
+		</parameter>
+		<parameter name="blink" type="boolean">
```

Comment:
Ok so just so i understand the correct implementation would be to have a blink channel that the user would turn off or on via an item?  Or would the channel then have a configuration for the blink settings to turn it off and on?  I guess my concern with the item is that then in order to have the setting persist between restarts the user then has to persist that item which just seems to add a layer of complexity. OR perhaps i am not fully understanding what needs to be done here.

This setting is really a device configuration setting where the user sets whether or not they want the physical device to blink its LED blue when it sends a WIFI command.  This seems like a device configuration that was being discussed by https://github.com/eclipse/smarthome/issues/3484#issuecomment-417612975  I take it from that discussion however that this has not been fully implemented yet.  However, as that is really what this setting would be I would prefer to implement the setting here in a way that would make the transition to a device configuration setting the smoothest for the user.

Reply:
>  the user would turn off or on via an item?

Yes, that's the idea. After all, it is a feature that you turn on and off on the device.

>  in order to have the setting persist between restarts

Does this mean that it isn't possible to read the state from the device?

Reply:
Ok, that's then a bit nasty indeed - a state that cannot be read, but set...
I assume the same holds true for the discovery setting?

So ok, I can live with having those as a configuration then. Let's hope that https://github.com/eclipse/smarthome/issues/3484 makes some progress soon as this would be the proper solution to use then.

Reply:
> Ok, that's then a bit nasty indeed - a state that cannot be read, but set...
> I assume the same holds true for the discovery setting?
> 
> So ok, I can live with having those as a configuration then. Let's hope that [eclipse/smarthome#3484](https://github.com/eclipse/smarthome/issues/3484) makes some progress soon as this would be the proper solution to use then.

correct what i can do is leave it as a configuration until https://github.com/eclipse/smarthome/issues/3484 is completed and then the binding could be altered to use that